<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutForm</class>
 <widget class="QWidget" name="AboutForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>540</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>About</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lblVersion">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>SYLoader</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Brought to you by Panagiotis Anastasiadis. SYLoader is dedicated to a special girl. Enjoy!</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Contributors</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="txtContributors"/>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="lbtnHomepage">
       <property name="text">
        <string>Visit Project's Page</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="lbtnContact">
       <property name="text">
        <string>Contact Developer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="lbtnLicense">
       <property name="text">
        <string>View GPL License</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Licensed under the terms and conditions of the GNU GPL v2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblOpenSSLNotice">
       <property name="text">
        <string>This product includes software developed by the OpenSSL Project for use in the OpenSSL Toolkit (http://www.openssl.org/)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
